app.js

```diff
@@ -4,9 +4,15 @@ const path = require('path');
 const cookieParser = require('cookie-parser');
 const logger = require('morgan');
 const mongoose = require('mongoose');
+const session = require('express-session');
+const passport = require('passport');
+const flash = require('connect-flash');
+const validator = require('express-validator');
+const MongoStore = require('connect-mongo')(session);
 
 //set up database connection
-mongoose.connect('mongodb://localhost:27017/chicken-fries',{ useNewUrlParser:true});
+mongoose.connect('mongodb://admin:root01@ds163402.mlab.com:63402/chicken-fries',{ useNewUrlParser:true});
```

Comment:
You'll want to avoid hard-coding paths in, and instead use environment variables with local fallbacks. Otherwise, when testing locally users would also be affecting the mlab server. It's also worth thinking about what happens to live if staging creates a bug that corrupts or changes data in the database. Since they share the same db, users of the live application would be affected.